014_sumOfSquare.js

```diff
@@ -0,0 +1,16 @@
+// 14. **Write program to check if number can be expressed as sum of squares of two whole numbers**
+
+const sumOfSquare = (number) => {
+
+    for (let i = 0; i <= number / 5; i++) {
```

Comment:
Wh y are you dividing the number by 5

Reply:
In the case of larger numbers, we need only up to 1/5 of the number to check the sum of the square.  so I put that.